gtsam/sfm/tests/CMakeLists.txt

```diff
@@ -1 +1,3 @@
-gtsamAddTestsGlob(sfm "test*.cpp" "" "gtsam")
+set(tests_excluded testMFAS.cpp testBinaryMeasurement.cpp)
+gtsamAddTestsGlob(sfm "test*.cpp" "${tests_excluded}" "gtsam")
+gtsamAddTestsGlob(sfm2 "testBinaryMeasurement.cpp testMFAS.cpp" "" "gtsam")
```

Comment:
Can you please explain what is the purpose of doing this?

Reply:
I define in my fix tests that are conflicting with each other. I think there is a bug / feature in Microsfot c++ compiler, that it cannot have the same definition of template function/class in several mains.